subsys/mgmt/mcumgr/Kconfig

```diff
@@ -131,6 +131,56 @@ config FS_MGMT_DL_CHUNK_SIZE
 
 endif
 
+config FS_MGMT_FILE_STATUS_ENABLED
+	bool "File status command enabled"
+	default y
+	help
+	  This command allows a remote device to retrieve the status of a file,
+	  at present only the size of the file is returned (if it exists).
+
+config FS_MGMT_CHECKSUM_HASH_SUPPORTED
+	bool "Checksum/hash mcumgr functions"
+	depends on HEAP_MEM_POOL_SIZE != 0
```

Comment:
This is very weak dependency, you can set the `HEAP_MEM_POOL_SIZE` to any non-0 and the functions will still not work because something else will take all of the heap.

Reply:
Changed to stack, default at 128